src/localization/locales/en-US.json

```diff
@@ -395,7 +401,116 @@
         "unique": "$t(terraso_api.models.{{model}}) {{field}} must be unique."
     },
     "gis": {
-        "openstreetmap_api_error": "Failed to request data from nominatim.openstreetmap.org API"
+        "openstreetmap_api_error": "Failed to request data from nominatim.openstreetmap.org API",
+        "map_search_placeholder": "Enter a city, town or region",
+        "map_layer_streets": "Streets",
+        "map_layer_satellite": "Satellite",
+        "map_draw": {
+            "draw": {
+                "toolbar": {
+                    "actions": {
+                        "title": "Cancel drawing",
+                        "text": "Cancel"
+                    },
+                    "finish": {
+                        "title": "Finish drawing",
+                        "text": "Close Shape"
+                    },
+                    "undo": {
+                        "title": "Delete last point drawn",
+                        "text": "Delete last point"
+                    },
+                    "buttons": {
+                        "polyline": "Draw a polyline",
+                        "polygon": "Draw a polygon",
+                        "rectangle": "Draw a rectangle",
+                        "circle": "Draw a circle",
+                        "marker": "Draw a marker",
+                        "circlemarker": "Draw a circlemarker"
+                    }
+                },
+                "handlers": {
+                    "circle": {
+                        "tooltip": {
+                            "start": "Click and drag to draw circle."
+                        },
+                        "radius": "Radius"
+                    },
+                    "circlemarker": {
+                        "tooltip": {
+                            "start": "Click map to place circle marker."
+                        }
+                    },
+                    "marker": {
+                        "tooltip": {
+                            "start": "Click map to place marker."
+                        }
+                    },
+                    "polygon": {
+                        "tooltip": {
+                            "start": "Click to start drawing shape.",
+                            "cont": "Click to continue drawing shape.",
```

Comment:
```suggestion
                            "start": "Start drawing shape.",
                            "cont": "Continue drawing shape.",
```

Reply:
@paulschreiber I am not sure if this is better, I think the message intention is to make it clear that they have to click, without that action seems like you are already perfomring the action, can you add more context on why you think this is better?

Reply:
"Click" (a) doesn't makes sense on mobile devices (b) is not accessible languages. It's also redundant.